rise/static/main.js

```diff
@@ -806,62 +818,56 @@ define([
       return metadata.slideshow;
     }
 
-    // accessing Jupyter.actions directly results in a warning message
-    // https://github.com/jupyter/notebook/issues/2401
-    let actions = Jupyter.notebook.keyboard_manager.actions;
-    actions.register(
-      {
-        help: '(un)set current cell as a Slide cell',
-        handler: function() {
-          let slideshow = init_metadata_slideshow();
-          slideshow.slide_type = (slideshow.slide_type == 'slide') ? '' : 'slide';
-	  Jupyter.CellToolbar.rebuild_all();
-        }
-      },
-      "toggle-slide", "RISE");
-
-    actions.register(
-      {
-        help: '(un)set current cell as a Sub-slide cell',
-        handler: function() {
-          let slideshow = init_metadata_slideshow();
-          slideshow.slide_type = (slideshow.slide_type == 'subslide') ? '' : 'subslide';
-          Jupyter.CellToolbar.rebuild_all();
-        }
-      },
-      "toggle-subslide", "RISE");
-
-    actions.register(
-      {
-        help: '(un)set current cell as a Fragment cell',
-        handler: function() {
-          let slideshow = init_metadata_slideshow();
-          slideshow.slide_type = (slideshow.slide_type == 'fragment') ? '' : 'fragment';
-	  Jupyter.CellToolbar.rebuild_all();
-        }
-      },
-      "toggle-fragment", "RISE");
-
-    actions.register(
-      { help: 'render all cells (all cells go to command mode)',
-        handler: function() {
-          Jupyter.notebook.get_cells().forEach(function(cell){
-	    cell.render();
-          })
-        }
-      },
-      "render-all-cells", "RISE");
-
-    actions.register(
-      {
-        help: 'edit all cells (all cells go to edit mode)',
-        handler: function() {
-          Jupyter.notebook.get_cells().forEach(function(cell){
-	    cell.unrender();
-          })
-        }
-      },
-      "edit-all-cells", "RISE");
+    // newtype can be any of 'slide' 'subslide' 'fragment' 'note' 'skip'
+    function toggle_slide_type(newtype) {
```

Comment:
this should be `new_type` OR change those references below to `newtype`, otherwise you will get an error...

Reply:
good catch, thanks for spotting that one - fixed in 72c8586